### PR TITLE
feat(networkfirewall): add new check `networkfirewall_policy_default_action_full_packets`

### PR DESCRIPTION
**Pull request by HugoPBrito**
### Context

In a network firewall policy, stateless actions are applied to packets without maintaining session state. This is crucial for efficiently handling high volumes of traffic, where the default action determines whether packets are allowed, dropped, or forwarded. Having the correct default action helps prevent unauthorized or unintended traffic from passing through the network firewall, which is especially critical in maintaining a secure network configuration.

### Description

This check ensures that the default stateless action for full packets in AWS Network Firewall policies is set to either `drop` or `forward`. It evaluates firewall configurations to prevent the default setting from being "pass," which could allow unintended traffic through the firewall. A proper configuration is essential to control how the firewall handles network traffic in a Virtual Private Cloud (VPC). If the default action is set to `pass`, the check fails, as this could expose the network to security risks by allowing unfiltered traffic.

### Checklist

- Are there new checks included in this PR? Yes.
    - If so, do we need to update permissions for the provider? Please review this carefully.
- [x] Review if the code is being covered by tests.
- [x] Review if code is being documented following this specification https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings
- [x] Review if backport is needed.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by HugoPBrito** [comment]
The name of this check is also open to suggestions as I'm not able to figure out a better one. Other ideas are:

1. `networkfirewall_policy_default_stateless_action_secure_full_packets`
2. `networkfirewall_policy_secure_full_packets`
3. `networkfirewall_policy_drop_or_forward_full_packets`

File: prowler/providers/aws/services/networkfirewall/networkfirewall_policy_default_action_drop_forward/networkfirewall_policy_default_action_drop_forward.metadata.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "Provider": "aws",
+  "CheckID": "networkfirewall_policy_default_action_drop_forward",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/networkfirewall/networkfirewall_policy_default_action_drop_forward/networkfirewall_policy_default_action_drop_forward.metadata.json]:
```suggestion
  "CheckID": "networkfirewall_policy_default_action_full_packets",
```

File: prowler/providers/aws/services/networkfirewall/networkfirewall_policy_default_action_drop_forward/networkfirewall_policy_default_action_drop_forward.metadata.json
Comment (MrCloudSec) [inline prowler/providers/aws/services/networkfirewall/networkfirewall_policy_default_action_drop_forward/networkfirewall_policy_default_action_drop_forward.metadata.json:1]: Please fill the title and description

File: prowler/providers/aws/services/networkfirewall/networkfirewall_policy_default_action_drop_forward/networkfirewall_policy_default_action_drop_forward.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            report = Check_Report_AWS(self.metadata())
+            report.region = firewall.region
+            report.resource_id = firewall.name
+            report.resource_arn = arn
+            report.resource_tags = firewall.tags
+            report.status = "FAIL"
+            report.status_extended = f"Network Firewall {firewall.name} policy does not drop or forward full packets by default."
+
+            for action in firewall.default_stateless_actions:
+                if action == "aws:drop" or action == "aws:forward_to_sfe":
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/networkfirewall/networkfirewall_policy_default_action_drop_forward/networkfirewall_policy_default_action_drop_forward.py]:
```suggestion
                if "aws:drop" in firewall.default_stateless_actions or "aws:forward_to_sfe" in firewall.default_stateless_actions:
```

File: prowler/providers/aws/services/networkfirewall/networkfirewall_policy_default_action_full_packets/networkfirewall_policy_default_action_full_packets.metadata.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+  "Provider": "aws",
+  "CheckID": "networkfirewall_policy_default_action_full_packets",
+  "CheckTitle": "NetworkFirewall firewall policy default action for fragmented packets is set to drop or forward.",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/networkfirewall/networkfirewall_policy_default_action_full_packets/networkfirewall_policy_default_action_full_packets.metadata.json]:
```suggestion
  "CheckTitle": "NetworkFirewall firewall policy default action for full packets is set to drop or forward.",
```

**Review by MrCloudSec — APPROVED** [review]

**Pull request merged**